src/explore-education-statistics-admin/src/pages/release/utils/releaseSummaryUtil.ts

```diff
@@ -11,6 +12,8 @@ export const getReleaseStatusLabel = (
       return 'In Review';
     case 'Approved':
       return 'Approved';
+    case isLive && 'Approved':
```

Comment:
Have you tested this? Doesn't seem like the branch would ever run because the 'Approved' branch above would be invoked first?

Would suggest having the `isLive` check at the top of the function, outside of the switch, as it should override everything else.